db/re/item_db.conf

```diff
@@ -49442,9 +49442,9 @@ item_db: (
 	Weight: 10
 	Loc: 2
 	Script: <"
-		bonus2 bSkillAtk,WM_REVERBERATION,50;
-		if (getequipweaponlv(EQI_HAND_R) == 4) bonus2 bSkillAtk,WM_REVERBERATION,50;
-		if (getrefine() >= 10) bonus2 bSkillAtk,WM_REVERBERATION,50;
+		bonus2 bSkillAtk,WM_REVERBERATION_MELEE,50;
```

Comment:
after testing it looks you're right it should increase the damage for the both hits the melee and the magic.